diff --git a/node_modules/js-yaml/dist/js-yaml.mjs b/node_modules/js-yaml/dist/js-yaml.mjs
index be71cad..fe209ac 100644
--- a/node_modules/js-yaml/dist/js-yaml.mjs
+++ b/node_modules/js-yaml/dist/js-yaml.mjs
@@ -3017,7 +3017,13 @@ function isPrintable(c) {
   return  (0x00020 <= c && c <= 0x00007E)
       || ((0x000A1 <= c && c <= 0x00D7FF) && c !== 0x2028 && c !== 0x2029)
       || ((0x0E000 <= c && c <= 0x00FFFD) && c !== CHAR_BOM)
-      ||  (0x10000 <= c && c <= 0x10FFFF);
+      ||  (0x10000 <= c && c <= 0x10FFFF)
+      || c === CHAR_TAB
+      || c === CHAR_LINE_FEED
+      || c === CHAR_CARRIAGE_RETURN
+      || c === 0x08
+      || c === 0x0B
+      || c === 0x0C;
 }
 
 // [34] ns-char ::= nb-char - s-white
